pkg/cstorvolumeclaim/v1alpha1/utils.go

```diff
@@ -0,0 +1,30 @@
+package v1alpha1
+
+import (
+	"encoding/json"
+	"fmt"
+
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	"k8s.io/apimachinery/pkg/util/strategicpatch"
+)
+
+// CVCKey returns an unique key of a CVC object,
+func CVCKey(cvc *apis.CStorVolumeClaim) string {
+	return fmt.Sprintf("%s/%s", cvc.Namespace, cvc.Name)
+}
+
+func getPatchData(oldObj, newObj interface{}) ([]byte, error) {
```

Comment:
is it required ?

Reply:
Yes it is being used in kubernetes.go file.